operator/pkg/central/extensions/reconcile_pvc_test.go

```diff
@@ -81,71 +81,40 @@ func notOwnedBy(central *platform.Central) pvcVerifyFunc {
 }
 
 func TestReconcilePVCExtension(t *testing.T) {
-	removedCentral := makeCentral(nil)
-	emptyNotDeletedCentral := makeCentral(nil)
-	emptyNotDeletedCentralWithDB := makeCentral(nil)
-	emptyNotDeletedCentralWithDB.Spec.Central.DB = &platform.CentralDBSpec{}
+	emptyCentral := makeCentral(nil)
+	emptyCentralWithDB := makeCentral(nil)
+	emptyCentralWithDB.Spec.Central.DB = &platform.CentralDBSpec{}
```

Comment:
What is the purpose of this line? It seems to replace a pointer to an empty struct with a pointer to another empty struct 🤔

Reply:
And if so, and this line is not needed, then we don't need both emptyCentral and emptyCentralWithDB right?

Reply:
I'm still not getting this 😅